a00_qpip/utils.py

```diff
@@ -90,7 +90,11 @@ def run_cmd(args, description="running a system command"):
             f"Encountered an error while {description} !",
             parent=iface.mainWindow(),
         )
-        message.setDetailedText(full_output)
+        if "no module named pip" in full_output.lower():
```

Comment:
Does this work on other locales? I guess this message is translated on non english systems?

Reply:
Yeah I know I though about it but I couldn't see any `tr()` (for e.g. f"Encountered an error while {description} !" on row 90) so I assumed that this only was targeting English, or?

Reply:
This message comes from the system, so it's out of our control.
I think we need to find another way of detecting that sip is missing, I would move this to a separate pull request.